.github/workflows/downstream_testing.yml

```diff
@@ -0,0 +1,110 @@
+name: downstream_testing_2
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      # Only run downstream tests on PRs for releases
+      - "[0-9]+.[0-9]+.x"
+  push:
+    branches:
+      - downstream_testing_2
```

Comment:
I took these guesses at when to run. The `on.push.branches` will of course have my branch removed.